gdx/src/com/badlogic/gdx/maps/tiled/tiles/AnimatedTiledMapTile.java

```diff
@@ -74,7 +74,33 @@ public int getCurrentFrameIndex() {
 
 		throw new GdxRuntimeException("Could not determine current animation frame in AnimatedTiledMapTile.  This should never happen.");
 	}
+	
+	public void restartAnimation(){
+		currentTime = lastTiledMapRenderTime % loopDuration;
+		timeOffset -= currentTime;
+	}
 
+	public void setFrame(int f){
```

Comment:
Please use a more meaningful name for this parameter, `frame` or `index` would be better than just `f`.